app/controllers/ubs_controller.rb

```diff
@@ -43,8 +43,22 @@ def cancel_appointment
     redirect_to ubs_index_path
   end
 
+  def cancel_all_future_appointments
+    appointments = future_appointments
+
+    appointments.update_all(active: false)
+
+    redirect_to ubs_index_path
+  end
+
   private
 
+  def future_appointments
+    @future_appointments = @ubs.appointments.where(
+      'start >= ? AND active = ?', Time.zone.now, true
+    )
+  end
```

Comment:
Maybe transform this into a UBS `scope`?
Like
```ruby
# in model/ubs.rb

scope :future_appointments -> do
  appointments.where('start >= ? AND active = ?', Time.zone.now, true)
end
```

Reply:
Tentemo mas falhemo